src/Restaurant.java

```diff
@@ -1,11 +1,11 @@
 import java.io.IOException;
-import java.util.Date;
+import java.time.LocalDateTime;
+import java.util.Iterator;
+import java.util.Scanner;
 
-public class Restaurant {
-	private static Order[] orderList = new Order[]{};
-	private int counter;
-	public char ans = 'y';
-	public static int i;
+public class Restaurant implements IRestaurant  {
+	private static Order[] orderList = new Order[10];
+	private Customer CustomerList[] = new Customer[10];
```

Comment:
variable names start with small letter.Change to:
```suggestion
	private Customer customerList[] = new Customer[10];
```